### PR TITLE
170: build: work around old cmake (<3.13)

### LINKED ISSUE #170: Check for cmake version to work around cmake < 3.13
Don't install subdirectory target in that case.

### PR DESCRIPTION
**Pull request by lifflander**
Fixes #170

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -144,8 +144,16 @@ if("${CMAKE_SOURCE_DIR}/src" STREQUAL PROJECT_SOURCE_DIR)
   message(STATUS "Checkpoint: building as top-level")
 else()
   message(STATUS "Checkpoint: building as a sub-directory")
-  install(TARGETS detector EXPORT ${CHECKPOINT_LIBRARY})
-  set(DETECTOR_EXPORT detector)
+
+  if(CMAKE_VERSION VERSION_LESS "3.13.0")
+    message(
+      WARNING "Checkpoint: Install targets skipped, effects packages built with a Checkpoint install"
```
Comment (PhilMiller) [inline src/CMakeLists.txt]: This warning should probably include a statement of the CMake version dependence and an issue reference in it

Comment (cz4rs) [inline src/CMakeLists.txt]: typo (?)
```suggestion
      WARNING "Checkpoint: Install targets skipped, affects packages built with a Checkpoint install"
```

Comment (cz4rs) [reply src/CMakeLists.txt]: ok, it's the same as in #1165

Comment (lifflander) [reply src/CMakeLists.txt]: Fixed

Comment (lifflander) [reply src/CMakeLists.txt]: Good catch, fixed!

**Pull request merged**